compliance/runners.py

```diff
@@ -180,7 +181,18 @@ def get_fetchers(self):
                         self.load_errors.add(load_err)
                 except AttributeError:
                     pass
-        return fetchers
+        if not (self.opts.include or self.opts.exclude):
+            return fetchers
+        include = {f'{f.__module__}.{f.__name__}' for f in fetchers}
+        if self.opts.include:
+            include = set(json.loads(open(self.opts.include).read()))
+        exclude = set()
+        if self.opts.exclude:
+            exclude = set(json.loads(open(self.opts.exclude).read()))
+        return filter(
```

Comment:
Why not another set operation here instead of the filter?

Reply:
I think the include - exclude would be done on each lambda invocation

Reply:
good point, I need to get this in ASAP.  You around for another review if I push the change which will negate the +1s?

Reply:
I'll add a follow up PR to patch.  Thanks for the suggestion.

Reply:
> Why not another set operation here instead of the filter? The intersection of fetchers and (include - exclude). I guess you want the fetcher class, not just it’s name.

Right, we want the class.

Reply:
> I think the include - exclude would be done on each lambda invocation - not a big deal, but might be worth doing it once

Handled in PR #86